setup.py

```diff
@@ -43,12 +43,10 @@ def read(*rnames):
         'Intended Audience :: Developers',
         'License :: OSI Approved :: Zope Public License',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.4',
-        'Programming Language :: Python :: 3.5',
```

Comment:
It would be good to also remove the unsupported Python versions from .travis.yml.

It would be even better to remove .travis.yml (which doesn't work anyway because the free credits ran out) and set up GitHub Actions.  We have templates and scripts in https://github.com/zopefoundation/meta/tree/master/config for automating that.

Reply:
ditto again for the when i have spare cycles.